tsl/org.testeditor.tsl.dsl/src/org/testeditor/tsl/dsl/GenerateTsl.mwe2

```diff
@@ -15,18 +15,20 @@ module org.testeditor.tsl.dsl.GenerateTsl
 import org.eclipse.emf.mwe.utils.*
 import org.eclipse.xtext.generator.*
 import org.eclipse.xtext.ui.generator.*
+import org.eclipse.xtext.xtext.generator.*
+import org.eclipse.xtext.xtext.generator.model.project.*
 
 var fileExtensions = "tsl"
 var projectName = "org.testeditor.tsl.dsl"
 var runtimeProject = "../${projectName}"
-var generateXtendStub = true
-var grammarURI = "platform:/resource/${projectName}/src/org/testeditor/tsl/dsl/Tsl.xtext"
+var rootPath = runtimeProject 
 
 Workflow {
 	
 	bean = StandaloneSetup {
```

Comment:
I think we don't need the `bean = StandaloneSetup` any longer with Xtext 2.9. Could you try to remove that one?